.babelrc

```diff
@@ -8,4 +8,4 @@
     "transform-es2015-destructuring",
     "transform-object-rest-spread"
   ]
-}
\ No newline at end of file
+}
```

Comment:
I do have editorconfig, but I reverted that file with the intellij git plugin and it looks like that plugin doesn't care about editorconfig :(